src/canmatrix/canmatrix.py

```diff
@@ -712,14 +716,7 @@ def to_compound_integer(self):
             return self.id
 
     def __eq__(self, other):  # type: (ArbitrationId) -> ArbitrationId
-        return (
```

Comment:
the original intention of _None_ was to create the possibility to compare an arbitration id while ignoring extended flag.
I think, there are use cases where people want to find a arbitration id 5 and they don't bother if its extended or regular

Reply:
What about `a.id == b.id` for that?

Reply:
I do feel like we need to remove None as a possible option for extended. Otherwise the following will be True which may cause issues during sorting, etc.:
Abr(extended=True) == Abr(extended=None) == Abr(extended=False) but Abr(extended=True) != Abr(extended=False)